### ISSUE TITLE
A Strange Historian - (5153)

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2107]
**In Progress Reports**

> The quest text could use some update, the inn location is missleading.

**Comment by Kestrana** [comment]
True, the quest text is now incorrect since we reverted Andorhol. The inn is now in the central southern part of town.
